### PR TITLE
Fix transferrable balance calculation

### PR DESCRIPTION
**Pull request by mmaurello**
### Description

Apply correct transferrableBalance formula
`free - max(0, frozen - reserved)`

### Checklist

- [x] If this requires a documentation change, I have created a PR that updates the `mkdocs/docs` directory
- [x] If this requires it, I have updated the Readme
- [x] If necessary, I have updated the examples
- [x] I have verified if I need to create/update unit tests
- [x] I have verified if I need to create/update acceptance tests
- [x] If necessary, I have run acceptance tests on this branch in CI

### CODE REVIEW

File: packages/builder/src/balance/BalanceBuilder.ts [hunk 1]
```diff
@@ -164,7 +164,10 @@ export async function calculateSystemAccountBalance(
   const balance = response.data as PalletBalancesAccountData &
     PalletBalancesAccountDataOld;
 
+  const free = BigInt(balance.free.toString());
```
Comment (Rihyx) [inline packages/builder/src/balance/BalanceBuilder.ts:167]: Can you please add some unit tests for this function?

**Review by Rihyx — CHANGES_REQUESTED** [review]
Please add some tests

File: packages/builder/src/balance/BalanceBuilder.ts (hunk 1 above)
Comment (mmaurello) [reply packages/builder/src/balance/BalanceBuilder.ts:167]: In `BalanceBuilder.test.ts` we have tests for `system.account`, which basically calls only this function. Do you want to write unit tests separately for the specific function?

Comment (Rihyx) [reply packages/builder/src/balance/BalanceBuilder.ts:167]: To be honest for this specific func I would like to have separate even if you generate a test using AI.

Comment (mmaurello) [reply packages/builder/src/balance/BalanceBuilder.ts:167]: added

**Review by Rihyx — APPROVED** [review]

**Pull request merged**